Sources/App/Routes/Routes.swift

```diff
@@ -1,23 +1,23 @@
 import Vapor
 
 extension Droplet {
-    func setupRoutes() throws {
-        get("hello") { req in
-            var json = JSON()
-            try json.set("hello", "world")
-            return json
-        }
-
-        get("plaintext") { req in
-            return "Hello, world!"
-        }
-
-        // response to requests to /info domain
-        // with a description of the request
-        get("info") { req in
-            return req.description
-        }
-
-        get("description") { req in return req.description }
+  func setupRoutes() throws {
```

Comment:
We need it?)

Reply:
Fixed (left only `hello` and `plaintext`, which we used in testing)

Reply:
Then we need comments about it, better to write special method for connection checking.